notebooks/idr0094-ic50.ipynb

```diff
@@ -6,11 +6,16 @@
    "source": [
     "# IC50 exploration\n",
     "\n",
-    "This notebook demonstrates how to process plates associated to the paper [A SARS-CoV-2 cytopathicity dataset generated by high-content screening of a large drug repurposing collection](https://doi.org/10.1038/s41597-021-00848-4).\n",
-    "We explore the metadata linked to [idr0094-ellinger-sarscov2](https://idr.openmicroscopy.org/search/?query=Name:idr0094).\n",
+    "This notebook demonstrates how to explore metadata associated to the paper [A SARS-CoV-2 cytopathicity dataset generated by high-content screening of a large drug repurposing collection](https://doi.org/10.1038/s41597-021-00848-4) and available at [idr0094-ellinger-sarscov2](https://idr.openmicroscopy.org/search/?query=Name:idr0094).\n",
     "\n",
-    "You can also explore the data and calculate the IC50 for each compound using the [Shiny app](https://binder.bioimagearchive.org/v2/gh/IDR/idr0094-ellinger-sarscov2/master?urlpath=shiny/apps/).\n",
-    "This notebook uses the Web API to load the data in R."
+    "To explore the metadata, few options are available:\n",
+    "\n",
+    "* Using the Web User Interface, you can download the [data as CSV](http://idr.openmicroscopy.org/webclient/omero_table/32361966/csv/?limit=9792)\n",
+    "* Using the Web API (this notebook), load the data and calculate the IC50\n",
+    "using a R library.\n",
+    "* Using [Shiny](https://binder.bioimagearchive.org/v2/gh/IDR/idr0094-ellinger-sarscov2/master?urlpath=shiny/apps/) app, you can explore the data and calculate the IC50\n",
+    "* Using the R gateway ([notebook](https://binder.bioimagearchive.org/v2/gh/IDR/idr0094-ellinger-sarscov2/master?urlpath=notebooks%2Fnotebooks%2Fidr0094-ic50_loadtable.ipynb%3FscreenId%3D2603)), you cna load the data and calculate the IC50.\n",
+    "* Other programming languages like Python, Java, Matlab."
```

Comment:
This sentence seems to be missing a verb. Maybe "You can also use other programming languages... " ?